### ISSUE TITLE
[assistant] Add suggestion module when detect bad sentences

### ISSUE THREAD

**Issue by stanlee974** [Orange-OpenSource/uuv#706]
## Introduction

We want to offer a LLM-based web accessibility assistant. It must analyze HTML code and must provide suggestions for correcting web accessibility. This tool will help project teams to comply with recognized accessibility standards such as the WCAG (Web Content Accessibility Guidelines), the RGAA (Référentiel Général d'Amélioration de l'Accessibilité) and the RAweb (Référentiel d'Accessibilité Web).

## Main usage
If we want to limit our checks to the block div. 
![element we want to check](https://github.com/user-attachments/assets/3339c92d-a933-4af6-8e6b-732013fa06ab)

The warning icon is displayed, it means that an accessibility error has been detected. 
We can display an icon to launch the suggestion and display it in the lower pane.

![poc](https://github.com/user-attachments/assets/db19fea9-b902-4ca2-8260-0e9784e101a5)

We can displayed theses fields on a suggestion area:

## HTML lacked field: 
```html 
<div>
 <h2>Académie Miro</h2>
 <p>Apprenez et développez vos compétences sur Miro</p>
</div> 
```

## field HTML field: 
```html 
<div aria-labelledby="academy-title">
 <h2 id="academy-title">Académie Miro</h2>
 <p>Apprenez et développez vos compétences sur Miro</p>
</div>
```

## Explanation field: 
```text
<Justifies the choice of proposal>
```

## WCAG rules
```text
<WCAG rules that are not in compliance>
```

## RAWEB rule
```text
<RGAA rule that is not in compliance>
```

## RGAA rule
```text
<RAWEB rule that is not in compliance>
```
